tests/e2e/testsLibrary/LanguageServerTests.ts

```diff
@@ -164,9 +162,10 @@ export class LanguageServerTests {
             } catch (err) {
                 // debug config is probably missing, refresh IDE and try again https://github.com/eclipse/che/issues/19887
                 await this.browserTabsUtil.refreshPage();
-                await this.driverHelper.wait(TimeoutConstants.TS_IDE_LOAD_TIMEOUT);
-                await this.ide.waitAndSwitchToIdeFrame();
-                await this.driverHelper.wait(TimeoutConstants.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM);
+                await this.ide.waitIde();
```

Comment:
let's remove commented code to keep script clean

Reply:
I will remove the commented lines

Reply:
Thanks.